fied/nei/nei_EF_calculations.py

```diff
@@ -1720,7 +1798,7 @@ def load_fueltype_dict(self):
             generic fuel types that have been applied to NEI data.
         """
 
-        with open('./tools/type_standardization.yml', 'r') as file:
+        with open(r'C:\Users\cskangos\Documents\GitHub\foundational-industry-energy-data\fied\tools\type_standardization.yml') as file:
```

Comment:
We should have a generic path here. Didn't appending the new tools path (on #L13) resolve accessing this file?

Reply:
For some reason the generic path is throwing an error even with the changes on #L13. The path exists and the .yml file is in the right spot but it is still not finding it. I will change it to the generic, but if there are any issues on your end if you try to run it and it donest find the path this may be a source of the issue

Reply:
I think I fixed this with bdeb541.